.travis.yml

```diff
@@ -29,7 +29,7 @@ before_install:
   - xcrun simctl list
   # Update the brew and build dependencies tools
   - brew outdated carthage || brew upgrade carthage
-  - Carthage update --platform iOS  --cache-builds
+  - Carthage update --platform iOS  --cache-builds --no-use-binaries
```

Comment:
You don't need this, you can pull the latest development (after the pr is merged)to pass the build

Reply:
removed